src/caches/firk_caches.jl

```diff
@@ -54,14 +55,16 @@ mutable struct RadauIIA5Cache{uType,cuType,uNoUnitsType,rateType,JType,W1Type,W2
   tol::Tol
   ηold::Tol
   nl_iters::Int
-  dtprev::Dt
   tmp::uType
   atmp::uNoUnitsType
   jac_config::JC
   linsolve1::F1
   linsolve2::F2
   rtol::rTol
   atol::aTol
+  dtprev::Dt
+  freshdt::Dt
```

Comment:
`freshdt` is supposed to mean the dt when `W` was formed, right? `fresh` implies that it's new, so that's really confusing. `Wdt` might be better, or `dt_at_W` or something like that.